src/app/drive/components/FileViewer/FileViewerWrapper.tsx

```diff
@@ -144,8 +145,7 @@ const FileViewerWrapper = ({ file, onClose, showPreview }: FileViewerWrapperProp
       downloadItem: onDownloadItemButtonClicked,
       moveToTrash: () => {
         if (isOwner) {
-          onMoveToTrashButtonClicked();
-          onClose();
+          onStopSharing?.();
```

Comment:
Why is there a function in the `moveToTrash` function that executes only `onStopSharing`? 
I think it would make more sense to change the name of the prop.